rebar.config

```diff
@@ -10,9 +10,9 @@
 
 {deps, [
         {blockchain, {git, "https://github.com/helium/blockchain-core.git",
-                      {branch, "master"}}},
+                      {branch, "andymck/poc-grpc-v2"}}},
```

Comment:
TODO - [ ] Revert branch ref when https://github.com/helium/blockchain-core/pull/1002 is merged.